src/functions/Add-PuppetReferenceDocumentation.ps1

```diff
@@ -0,0 +1,40 @@
+function Add-PuppetReferenceDocumentation {
+    <#
+      .SYNOPSIS
+        Generate REFERENCE.md
+      .DESCRIPTION
+        Generate REFERENCE.md
```

Comment:
```suggestion
        Generate REFERENCE.md file for the Puppet module from the auto-generated types for each DSC
        resource. This will *always* have the syntax but **may not** have the property documentation,
        depending on whether or not those reference docs were discoverable for each DSC resource.
```